samples/highcharts/series-dumbbell/lowmarker/demo.css

```diff
@@ -0,0 +1,5 @@
+#container {
+    height: 500px;
+    max-width: 600px;
+    margin: 0 auto;
```

Comment:
Metter apply max-width and margin to the figure element, so that the description is aligned with the chart.